### ISSUE TITLE
About the pred_camera_scale 

### ISSUE THREAD

**Issue by KaifuZ** [mkocabas/VIBE#86]
perspective_camera = torch.stack([pred_camera_translation_x, pred_camera_translation_y, 
                               2 * focal_length / (crop_size * pred_camera_scale + 1e-9)], dim=-1) 
Hello！Could you tell me what is the definition of pred_camera_scale？I am confused with (2 * focal_length / (crop_size * pred_camera_scale) , What do focal_length*2 and crop_size * pred_camera_scale mean?
Thx!

**Comment by longbowzhang** [comment]
Same confusion here.

1. As mentioned in the HMR that scale _s_ is essentially _focal_length/z_, but the following line https://github.com/mkocabas/VIBE/blob/07fa8d3463c4240e0199239a4d162e8bae489f25/lib/models/spin.py#L312 suggests that _0.5 * img_size(=224)_ comes into play, why?

2. This line code https://github.com/mkocabas/VIBE/blob/07fa8d3463c4240e0199239a4d162e8bae489f25/lib/models/spin.py#L345 suggests that points and translation, which is `translation = torch.stack([pred_camera[:, 1],
                              pred_camera[:, 2],
                              2 * 5000. / (224. * pred_camera[:, 0] + 1e-9)], dim=-1)`, are in the some but what space?

Thus, could you elaborate a little bit on the parameters of this weak perspective projection `pred_camera`?

Thanks in advance.
